### DISCUSSION TITLE
Gtk with wgpu

### DISCUSSION THREAD

**Discussion by wusyong** [gfx-rs/wgpu#2717] [Q&A]
Gtk has GlArea widget that can create OpenGL or an OpenGL ES context.
And looks like OpenGL ES is one of wgpu's backend.
Did anyone tried to create a gtk window with wgpu as example?
Or is it possible in theory?

**Comment by ThrashAbaddon** [comment]
I'm also interested in this. Maybe someone can give more details about this to get the fire started. 😃

**Comment by Beryesa** [comment]
Maybe it could be added as a separate backend like the new renderers?
https://blog.gtk.org/2024/01/28/new-renderers-for-gtk/
